common/src/main/java/io/netty/util/AsyncMapping.java

```diff
@@ -18,11 +18,11 @@
 import io.netty.util.concurrent.Future;
 import io.netty.util.concurrent.Promise;
 
-public interface AsyncMapping<IN, OUT> {
+public interface AsyncMapping<U, R> {
```

Comment:
Strange, I can't reply to the `U` vs. `IN` comment. Anyways, I use `U` in `@FunctionalInterface` for **U**ser provided arguments and `R` is the return type. I'll revert for a cleaner focused PR.